packages/guides-restructured-text/src/RestructuredText/DependencyInjection/ReStructuredTextExtension.php

```diff
@@ -55,4 +65,36 @@ public function process(ContainerBuilder $container): void
     {
         (new TextRolePass())->process($container);
     }
+
+    public function getConfigTreeBuilder(): TreeBuilder
```

Comment:
Do we want this in the rest extension or the code extension?

The language labels related closely to highlighting aliases, which are configured in the code extension. If we move the tabbed directive to reSt/core, I think we can move configuration block to guides-code (for normal use-cases, you can use the tabbed directive and for code specific docs you can use the guides-code extension).

Reply:
We do not use the code extension but have our own code highlighting. We would however still like to use this directive.

Reply:
Let me check that, because phpDocumentor is also not using the code extension. 

Maybe that extension should be more configurable to make this work. I like the idea to group this functionality as both directives focus on code.

Reply:
I have been thinking about this, the directive is related to the RST format, so it would be strange to put that in the code extension which is not bound to a input format. 

I'm still thinking about a way to make the configuration less of a thing... maybe we should create some kind of mapping between the code extension and the rst extension to make the configuration less of a hassle.

Reply:
It is in the rest extension now so that should be fine?